src/MoBi.Presentation/Tasks/Edit/EditTaskForContainer.cs

```diff
@@ -86,6 +94,66 @@ public override void Save(IContainer container)
          exportContainer(container, fileName);
       }
 
+      private void addIndividualParametersToContainers(IndividualBuildingBlock individual, IContainer container)
```

Comment:
would rename to `addIndividualParametersToContainer` (singular)
(and before rename the current `addIndividualParametersToContainer` to `addIndividualParametersToContainerByPath` or something like that)

Reply:
I'm going with ```addIndividualParametersToContainerAndSubContainers``` for this one and ```addIndividualParametersToContainerByPath ``` for the other